src/block_service/hdfs/hdfs_service.cpp

```diff
@@ -46,13 +46,20 @@ DSN_TAG_VARIABLE(hdfs_read_batch_size_bytes, FT_MUTABLE);
 DSN_DEFINE_uint32("replication", hdfs_read_limit_rate_megabytes, 200, "hdfs read limit(MB/s)");
 DSN_TAG_VARIABLE(hdfs_read_limit_rate_megabytes, FT_MUTABLE);
 
+DSN_DEFINE_uint32("replication", hdfs_write_limit_rate_megabytes, 200, "hdfs write limit(MB/s)");
```

Comment:
How about naming it like `hdfs_write_limit_mb_per_sec`, I think it would be self-explanatory and more readable. The same to `hdfs_read_limit_rate_megabytes`

Reply:
Ok

Reply:
`hdfs_read_limit_rate_megabytes` has been introduced a long time ago, and has been released in 2.2 and 2.3, keep in mind that renaming will break compatibility.

Reply:
I will make notes about config changes in this PR.